### ISSUE TITLE
discrepancy in `get_distance_matrix` doc-string

### ISSUE THREAD

**Issue by bobcolner** [hudson-and-thames/mlfinlab#474]
**Describe the bug**
discrepancy in `get_distance_matrix` doc-string. The 'absolute_angular' is reference in the doc-string but the function expects 'abs_angular'.

From the doc-string:
List of supported distance metrics to use for generating the distance matrix: ``angular``, ``squared_angular``,
and ``absolute_angular``.

from the source code:
```
if distance_metric == 'angular':
        distfun = lambda x: ((1 - x).round(5) / 2.) ** .5
    elif distance_metric == 'abs_angular':
        distfun = lambda x: ((1 - abs(x)).round(5) / 2.) ** .5
    elif distance_metric == 'squared_angular':
        distfun = lambda x: ((1 - x ** 2).round(5) / 2.) ** .5
    else:
        raise ValueError(f'{distance_metric} is a unknown distance metric. Please use one of the supported methods \
                            listed in the docsting.')
```

**Comment by PanPip** [comment]
Thank you for noticing this @bobcolner!

This bug will be fixed in the new MlFinLab version.

**Comment by PanPip** [comment]
This issue is now fixed in MlFinLab v 0.15.1.
